pipeline/builder/builder.go

```diff
@@ -312,6 +312,45 @@ func (b *Builder) buildDeployEmbeddedManifestStage(index int, s config.Stage) (*
 			return nil, errors.Wrapf(err, "could not parse manifest file: %s", file.File)
 		}
 
+		for i, obj := range objs {
+			u, ok := obj.(*unstructured.Unstructured)
+			if !ok {
+				return nil, errors.New("manifest parser returned an unexpected object type")
+			}
+			if u.GetKind() != "Deployment" {
+				continue
+			}
+
+			// if deployment set container overrides
+			var d v1beta1.Deployment
```

Comment:
The v1beta1 api extension for Deployment has actually been deprecated in k8s 1.16. We need to use appsv1: https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/